src/fsharp/Fsc/Fsc.fsproj

```diff
@@ -30,23 +31,9 @@
       <InProject>false</InProject>
     </FilesToLocalize>
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="InternalsVisibleTo.fs" />
-    <Compile Include="..\..\assemblyinfo\assemblyinfo.fsc.exe.fs">
-      <Link>Resources/assemblyinfo.fsc.exe.fs</Link>
-    </Compile>
-    <Compile Include="..\fscmain.fs">
-      <Link>fscmain.fs</Link>
-    </Compile>
-    <None Include="fsc.exe.config">
-      <Link>fsc.exe.config</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-    <None Include="..\..\windows\default.win32manifest">
-      <Link>default.win32manifest</Link>
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </None>
-  </ItemGroup>
+
+  <Import Project="Fsc.Sources.proj" />
```

Comment:
It turns out the F# project system in Visual Studio doesn't understand an "Import" of a .proj containing an ItemGroup.  Sad I know.  You get this in the solution explorer:
![image](https://user-images.githubusercontent.com/7204669/27671957-a5635734-5c8f-11e7-8a75-0476405354b2.png)

I think we will just have to duplicate the source file lists once again....